source/extinst.amd-gcn-shader.grammar.json

```diff
@@ -0,0 +1,23 @@
+{
+  "revision" : 2,
+  "instructions" : [
+    {
+      "opname" : "CubeFaceIndexAMD",
+      "opcode" : 1,
+      "operands" : [
+        { "kind" : "IdRef", "name" : "'P'" }
+      ]
```

Comment:
Hmm.  The grammar file documentation only describes the "extensions" field for ValueType enumerants.  Seems it should be added for Instruction objects and BitEnum as well.

I'll add the field for this PR anyway.